squad-server/plugins/discord-killfeed.js

```diff
@@ -82,11 +82,6 @@ export default class DiscordKillFeed extends DiscordBasePlugin {
           ? `[${info.victim.steamID}](https://steamcommunity.com/profiles/${info.victim.steamID})`
           : 'Unknown',
         inline: true
-      },
-      {
```

Comment:
Why is this removed? Will need to be added back in if you want this PR approved.

Reply:
To quote myself,

> In addition, removes a redundant code in `DiscordKillFeed` class that used to cause an access error, where if `info.victim` was `null` then the field access statement `info.victim.name` would fail, causing SquadJS to crash.

I observed this behavior randomly in my own dedicated server while the code on `master` branch was running. It would throw access error while trying to access `info.victim.name` when `info.victim` was `null`.

After removing this bit, I can confirm that I haven't encountered that error ever since. Also, if you carefully check that list variable, there is already a duplicate object:
```
      {
        name: "Victim's Name",
        value: info.victim ? info.victim.name : 'Unknown',
        inline: true
      },
```

I hope that was enough reason.

Reply:
That isn't a reason to pull it off. It needs to be corrected as there are users that needs to use that info. The crash will need to be corrected elsewhere in the code.

Reply:
Also each change needs to have its own PR. This change isn't tied at all to the main core issue/correction of this PR. That's why I accidentally overlooked it.

Reply:
done

Reply:
Although I still do not understand how that is not a duplicate object. The same field is literally defined 2 positions behind except with a value that doesn't fail in case `info.victim` is `null` 🤷 
```typescript
    const fields = [
      {
        name: "Attacker's Name",
        value: info.attacker.name,
        inline: true
      },
      {
        name: "Attacker's SteamID",
        value: `[${info.attacker.steamID}](https://steamcommunity.com/profiles/${info.attacker.steamID})`,
        inline: true
      },
      {
        name: "Attacker's EosID",
        value: info.attacker.eosID,
        inline: true
      },
      {
        name: 'Weapon',
        value: info.weapon
      },
      {
        name: "Victim's Name",
        value: info.victim ? info.victim.name : 'Unknown',
        inline: true
      },
      {
        name: "Victim's SteamID",
        value: info.victim
          ? `[${info.victim.steamID}](https://steamcommunity.com/profiles/${info.victim.steamID})`
          : 'Unknown',
        inline: true
      },
      {
        name: "Victim's Name",
        value: info.victim.name,
        inline: true
      }
    ];
```

Reply:
While your fix is meaningful in this particular case (I can only assume there was a "bad" merge that caused the duplication) werewolf asks to keep killfeed fix separate from this specific issue simply for clarity. I am sure that if you commit this fix as a separate PR (and providing the rationale from above) it will be merged in.